chapters/io/index.rst

```diff
@@ -0,0 +1,319 @@
+How to handle Input and Output for Flows
+=======================================
+
+This chapter explains how to provide input data to the `Flow API`.
+Moreover, it describes how to handle the output via callback functions.
```

Comment:
I think you can shorten this to one sentence? 

This chapter explains how to input data via `Flow API`and handle outputs via callback functions.

Reply:
Changed it to:
This chapter explains how input and output data is handled by the `Flow API`.